scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -275,6 +275,16 @@
 </div>
 {% endmacro %}
 
+{% macro variant_size_filter(form) %}
```

Comment:
I've created this macro to avoid repetitions as much as possible. I'm not sure how to improve so that repeated code is <3% repeated.